README.md

```diff
@@ -52,6 +52,7 @@ Usage: cargo info [OPTIONS] <SPEC>
 Options:
       --index <INDEX>        Registry index URL to search packages in
       --registry <REGISTRY>  Registry to search packages in
+      --path <PATH>          Filesystem path to local crate to inspect
```

Comment:
Is our intention to support all sources (registry, path, git)?

`cargo search` only supports registries.  `cargo add` supports different source types.

Reply:
Yes. I think because we already supported local workspace, it might be better to support path and git sources.